### ISSUE TITLE
Try out DigitalOcean's block storage with Kontena

### ISSUE THREAD

**Issue by jkesanie** [ATTX-project/platform-deployment#50]
#### Description
_Describe the issue or task at hand with dependencies and possible links to code/external/internal sources._
#### DoD
_Itemized things or artifacts of the task at hand, what one expects to achieve - tangible output._
#### Testing
_How one would test that the things/artifacts have been achieved. Options (Peer review, Unit Test, BDD testing, Integration Test)._

**Comment by joasilva** [comment]
Results to be appended to Kontena report on Gitbook as well (https://github.com/ATTX-project/platform-deployment/issues/51).

**Comment by joasilva** [comment]
Kontena node created in Digital Ocean as part of the attx-test grid. Exploring block storage configuration with DO.

**Comment by joasilva** [comment]
DO block storage volume mounted to new Kontena node and used for MySQL storage.

**Comment by joasilva** [comment]
Done. Steps:

1. Added kontena node in DigitalOcean:
`kontena digitalocean node create`

2. Attached block storage to newly kontena node in DigitalOcean UI

3. Configured and mounted block storage device trough kontena-cli:
```
$ kontena node ssh <node_name>
$ sudo mkfs.ext4 -F /dev/disk/by-id/scsi-0DO_Volume_volume-fra1-01
$ sudo mkdir -p /mnt/volume-fra1-01; sudo mount -o discard,defaults /dev/disk/by-id/scsi-0DO_Volume_volume-fra1-01 /mnt/volume-fra1-01; echo /dev/disk/by-id/scsi-0DO_Volume_volume-fra1-01 /mnt/volume-fra1-01 ext4 defaults,nofail,discard 0 0 | sudo tee -a /etc/fstab
```

Edited mysql service in attx stack yml file to use the block storage device mount point:
```
    volumes:
      - /mnt/volume-fra1-01:/var/lib/mysql
```
4. Deployed the stack:
`$ kontena stack install --name attx attx-do-kontena.yml`

5. UV UI available at "http://207.154.202.178:8080/unifiedviews/" and MySQL files at block storage device:

```
ng-morning-64 volume-fra1-01 # ls -l
total 176200
-rw-rw----. 1 999 rkt-admin    16384 Aug  3 13:54 aria_log.00000001
-rw-rw----. 1 999 rkt-admin       52 Aug  3 13:54 aria_log_control
-rw-rw----. 1 999 rkt-admin 50331648 Aug  3 14:21 ib_logfile0
-rw-rw----. 1 999 rkt-admin 50331648 Aug  3 13:53 ib_logfile1
-rw-rw----. 1 999 rkt-admin 79691776 Aug  3 14:21 ibdata1
drwx------. 2 999 rkt-admin    16384 Aug  3 12:05 lost+found
-rw-rw----. 1 999 rkt-admin        0 Aug  3 13:53 multi-master.info
drwx------. 2 999 rkt-admin     4096 Aug  3 13:53 mysql
drwx------. 2 999 rkt-admin     4096 Aug  3 13:53 performance_schema
-rw-rw----. 1 999 rkt-admin    24576 Aug  3 13:54 tc.log
drwx------. 2 999 rkt-admin     4096 Aug  3 13:54 unified_views

```
